src/plugins/kibana/public/management/styles/main.less

```diff
@@ -41,6 +41,30 @@ kbn-management-app {
 
 kbn-management-landing {
   padding: 5px;
+}
+
+  .management-panel-product {
```

Comment:
By un-nesting these selectors we pave the way for refactoring them into reusable components in the (near?) future. Un-nesting them reduces specificity and makes it easier to move them around while being confident we're not breaking anything.